sonic-net/sonic-buildimage
# [Marvell] Update armhf driver version

Changes in MRVL_PRESTERA_DRIVER_1.4:
- Memory leak fixed by adding missing releasing pci device.
- Fixes for 5.10 kernel porting.

Change-Id: I1d7ee4ec02ec17a29ddb8473725ab68ca399748b



#### Why I did it

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

#### How to verify it
Tested by building master sonic-marvell-armhf.bin and loaded on switch to check the 'show interface status' for port up.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
